apps/mobile/src/components/action-bar/action-bar-container.tsx

```diff
@@ -125,94 +130,84 @@ export function useActionBarContext() {
 interface ActionBarButtonProps {
   onPress: () => void;
   icon: ReactNode;
-  label: string;
+  label?: string;
   testID?: string;
 }
 function ActionBarButton({ onPress, icon, label, testID }: ActionBarButtonProps) {
   return (
-    <Pressable
-      onPress={onPress}
-      justifyContent="center"
-      alignItems="center"
-      flex={1}
-      height="100%"
-      flexDirection="row"
-      gap="2"
-      pressEffects={legacyTouchablePressEffect}
-      haptics="soft"
-      testID={testID}
-    >
-      {icon}
-      <Text variant="label02">{label}</Text>
-    </Pressable>
+    <Box flex={1} flexDirection="row" justifyContent="center" alignItems="center">
+      <Pressable
+        onPress={onPress}
+        justifyContent="center"
+        alignItems="center"
+        flex={1}
+        height="100%"
+        flexDirection="row"
+        gap="2"
+        pressEffects={legacyTouchablePressEffect}
+        haptics="soft"
+        testID={testID}
+      >
+        {icon}
+        {label && <Text variant="label02">{label}</Text>}
+      </Pressable>
+    </Box>
   );
 }
 
 export const ActionBarContainer = forwardRef<ActionBarMethods>((_, ref) => {
   const { sendSheetRef, receiveSheetRef } = useSheetNavigatorContext();
   const wallets = useWallets();
   const addWalletSheetRef = useRef<SheetRef>(null);
+  const browserSheetRef = useRef<SheetRef>(null);
+  const browserRef = useRef<SheetRef>(null);
+  const releaseBrowserFeature = useReleaseBrowserFeatureFlag();
+
+  useOnMount(() => {
+    browserRef.current?.present();
+  });
 
   const actionBar = isEmptyArray(wallets.list) ? (
-    <ActionBar
-      ref={ref}
-      center={
-        <ActionBarButton
-          onPress={() => addWalletSheetRef.current?.present()}
-          icon={<PlusIcon />}
-          label={t({
-            id: 'action_bar.add_wallet_label',
-            message: 'Add Wallet',
-          })}
-          testID={TestId.homeAddWalletButton}
-        />
-      }
-    />
+    <ActionBar ref={ref}>
+      <ActionBarButton
+        onPress={() => addWalletSheetRef.current?.present()}
+        icon={<PlusIcon />}
+        label={t({
+          id: 'action_bar.add_wallet_label',
+          message: 'Add Wallet',
+        })}
+        testID={TestId.homeAddWalletButton}
+      />
+    </ActionBar>
   ) : (
-    <ActionBar
-      ref={ref}
-      left={
+    <ActionBar ref={ref}>
+      <ActionBarButton onPress={() => sendSheetRef.current?.present()} icon={<PaperPlaneIcon />} />
+      <ActionBarButton onPress={() => receiveSheetRef.current?.present()} icon={<InboxIcon />} />
+      {isDev() && (
         <ActionBarButton
-          onPress={() => sendSheetRef.current?.present()}
-          icon={<PaperPlaneIcon />}
-          label={t({
-            id: 'action_bar.send_label',
-            message: 'Send',
-          })}
+          onPress={() => {
```

Comment:
Lets just get rid of this button if it's not doing anything? Who knows when we will add `Swap`